app/models/invoice.rb

```diff
@@ -10,6 +10,18 @@ def self.find_from(customer_id)
     where(customer_id: customer_id)
   end
 
+  def status_view_format
+    if status == "in_progress"
+      "In Progress"
+    else
+      status.capitalize
+    end
+  end
+
+  def created_at_view_format
```

Comment:
I think this method isn't needed if the view can at least access the model's `created_at`. It should be within the view's responsibility to format the raw date it gets from the model.

Reply:
I had the same EXACT thought process. I had it in the view for Admin Dashboard, then when I needed it on Admin Invoice Show page I thought it would be best to DRY it up.

Reply:
I am also thinking these could potentially live in a module or different class that all the models can utilize